tests/pytests/test_newapi.py

```diff
@@ -187,3 +187,78 @@ def check_result(method, check_json, check_data):
     check_result(session.bulk_read_json, True, False)
     check_result(session.bulk_read_data, False, True)
     check_result(session.bulk_read, True, True)
+
+
+@pytest.mark.usefixtures('servers')
+def test_session_bulk_remove(simple_node):
+    """Test bulk_remove."""
+    session = elliptics.newapi.Session(simple_node)
+    session.trace_id = make_trace_id('test_session_bulk_remove')
+    session.groups = session.routes.groups()
+    session.set_filter(elliptics.filters.all_with_ack);
+
+    # prepare test data
+    keys = []
+    datas = {}
+    def make_item(data, json):
+        return {"data": data, "json": json}
+    groups = session.groups;
+    def prepare_data():
+        del keys[:]
+        datas = {}
+        write_results = []
+        for group_id in groups:
+            session.groups = [group_id]
+            for i in range(10):
+                eid = session.transform('k_br{}'.format(i))
+                eid.group_id = group_id
+                keys.append(eid)
+                data = "data{}_{}".format(group_id, i)
+                json_string = json.dumps({'some': "json{}_{}".format(group_id, i)})
+                datas[repr(eid)] = make_item(data, json_string)
+                result = session.write(eid, json_string, len(json_string), data, len(data))
+                write_results.append(result)
+
+        for r in write_results:
+            assert r.get()[0].status == 0
+
+        assert len(keys) == len(datas)
+
+    prepare_data()
+
+    #remove data
+    cur_ts = session.get_timestamp()
```

Comment:
By default `timestamp` in session isn't filled, so here `cur_ts` will be equal to `elliptics.Time(0xffffffffffffffff, 0xffffffffffffffff)`.